### DISCUSSION TITLE
ALE doesn't work now, but it worked previously

### DISCUSSION THREAD

**Discussion by jck307** [w0rp/ale#3877] [Q&A - Ask for help with problems]
I was messing around with ale configuration and now it doesn't really work. I've reinstalled it, called :UpdateRemotePlugins, run commands that I've hoped would've fixed it (:ALEEnable, :ALEDisable, :ALEReset, :ALELint) but none of them do anything.

I've even used -u (config file) as an argument with an empty file, manually installing ale with vim-plug, loaded a python file and there's no difference.

Sometimes it showed a red underline below some errors but no sign gutter or anything when interacting with the underlined word. What i mean by sometimes is, yes, sometimes. And i don't know why only sometimes.

I tried disabling options i set to ALE in my vimrc and i restarted vim multiple times.

When it worked i hadn't installed any linter manually (although they may have already been installed), but now it doesn't work and i don't think I've uninstalled anything. I've now also tried installing linters with pip, the python package installer, but it still doesn't worked. And I've restarted vim.

So yeah ALE doesn't work and i don't know why. I would just like any tip on what may cause this.

[Full :ALEInfo output with some added information at the end](https://pastebin.com/raw/VKrGd3nn)

**Comment by hsanson** [comment]
You have all linters disabled:

> Enabled Linters: []

Make sure to properly set the `g:ale_linters` variable. This is your current config according to ALEInfo:

> let g:ale_linters = {'python': []}

To enable linters for python change add the linters you have installed in the array like below:

> let g:ale_linters = { 'python': ['flake8', 'pylint', 'pyright'] }
